sdk/core/core-https/src/policies/disableResponseDecompressionPolicy.browser.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/*
+ * NOTE: When moving this file, please update "browser" section in package.json
```

Comment:
Much appreciated note.

Reply:
this was in the original file! credit goes to @jeremymeng for it